docs/sources/write/front-matter/index.md

```diff
@@ -68,38 +69,65 @@ The correct way to use aliases depends on whether the project is versioned or no
 
 #### Versioned projects
 
-Do not include an `aliases` entry that refers to the initial published website directory.
-The version in the URL path can cause undesirable redirects, such as a redirect from latest content to an old version.
-Aliases must be relative and not absolute paths so that old versions do not steal redirects from "latest" content when it is moved around.
+Aliases must be relative to avoid redirecting latest content to old versions.
+This happens when a page no longer exists in latest documentation but an alias with the latest version exists in old documentation.
```

Comment:
These two lines (72 -3) are confusing.  If a page no longer exists, and the alias is in an older version pointing to latest, wouldn't that result in a 404?

Reply:
Yeah I've found it hard to write this out.

Perhaps:

```suggestion
Aliases must be relative to avoid redirecting latest content to old versions.


If there is a page in the old documentation that has an alias that includes the version "latest", and the page referred to by that alias doesn't exist in the actual latest documentation, then Hugo creates a redirect at the page referred to by the alias.

That redirect redirects the user from latest documentation into the old documentation.
```

Reply:
Your revision makes a whole lot more sense!